packages/components/pie-webc/test/componentService.spec.js

```diff
@@ -1,3 +1,4 @@
+import path from 'path';
```

Comment:
Is this import needed in the test?

Reply:
It's there to simplify the "mock" implementation for `path.parse`, otherwise its usage would break tests.